distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -0,0 +1,298 @@
+
+==== Keystore and Certificate Management
+
+${branding} uses certificates in three ways:
+
+. Ensuring the privacy and integrity of message sent or received over a network.
+. Transmitting and receiving encrypted messages.
+. Authenticating an incoming user request.
+
+For more certificate configuration options, see <<_configuring_${ddf-security}, Configuring ${ddf-security}>>.
+
+===== Keystore Management
+
+Certificates, and sometimes their associated private keys, are stored in keystore files.
+${branding} includes two default keystore files, the server key store and the server trust store.
+The server keystore holds the certificates and private keys that ${branding} uses to identify itself to other nodes on the network.
+The truststore holds the certificates of nodes or other entities that ${branding} needs to trust.
+
+Certificates (and certificates with keys) can be managed in the ${admin-console}. Navigate to the ${ddf-security}application and select it.
+After it opens, select the notebook tab labeled "Certificates".
+This view shows the alias (name) of every certificate in the trust store and the key store.
+It also displays if the entry includes a private key ("Is Key") and the encryption scheme (typically "RSA" or "EC").
+
+This view allows administrators remove certificates from ${branding}'s key and trust stores.
+It also allows administrators to import certificates and private keys into the keystores with the "+" button.
+The import function has two options: import from a file or import over HTTPS.
+The file option accepts a Java Keystore file or a PKCS12 keystore file.
+Because keystores can hold many keys, the import dialog asks the administrator to provide the alias of the key to import.
+Private keys are typically encrypted and the import dialog prompts the administrator to enter the password for the private key.
+Additionally, keystore files themselves are typically encrypted and the dialog asks for the keystore ("Store") password.
+
+The name and location of the ${branding} trust and key stores can be changed by editing the system properties files, `etc/system.properties`.
+Additionally, the password that ${branding} uses to decrypt (unlock) the key and trust stores can be changed here.
+
+IMPORTANT:
+====
+${branding} assumes that password used to unlock the keystore is the same password that unlocks private keys in the keystore.
+====
+
+. Set the keystore and truststore Java properties
```

Comment:
I think line 40 needs a little more. How about? 

"The location, file name, passwords and type of the server and trust key stores can be set in the system.properties file"

Then you go delete lines 51 and 52.